using-Try-except.py

```diff
@@ -0,0 +1,4 @@
+try:
+  raise Exception("Hello, World!")
+except Exception as e:
+  print(str(e))
```

Comment:
There is an existing implementation for this solution that you've submitted!

You can find the existing solution [here](https://github.com/Goku-kun/1000-ways-to-print-hello-world-in-python/blob/master/using-exception.py). The concept is the same (to use exceptions) so I'll be needing you to remove this submission from your PR.